src/plugins/blocksmith/components/instagram-media-block-modal/index.jsx

```diff
@@ -104,13 +102,13 @@ export default class InstagramMediaBlockModal extends React.Component {
   handleChangeTextarea(event) {
     let { errorMsg, hideCaption, id, isValid } = this.state;
     const input = event.target.value;
-    const isValidUrl = INSTAGRAM_MEDIA_REGEX.test(input);
+    const isValidUrl = getInstagramMediaId(input);
```

Comment:
@jalopez13 could just call this fn once, id = getBlah, instead of doing it again below

Reply:
@efjacobson 

Updated to only be called once.